pkg/printers/internalversion/printers_test.go

```diff
@@ -829,29 +829,19 @@ func TestPrintNodeStatus(t *testing.T) {
 			node: api.Node{
 				ObjectMeta: metav1.ObjectMeta{
 					Name:   "foo10",
-					Labels: map[string]string{"node-role.kubernetes.io/master": "", "kubernetes.io/role": "node", "kubeadm.alpha.kubernetes.io/role": "node"},
+					Labels: map[string]string{"node-role.kubernetes.io/master": "", "node-role.kubernetes.io/proxy": "", "kubernetes.io/role": "node"},
 				},
 			},
-			status: "Unknown,Master",
+			status: "master,node,proxy",
```

Comment:
Since we have added a new column named `"Role"`, these testcases should be moved to a new test `TestPrintNodeRole`. They do not belong to `Status` any more.